AeonCore/main.py

```
@@ -0,0 +1,514 @@
+        coherence_check = np.dot(state, intent) / (np.linalg.norm(state) * np.linalg.norm(intent) + 1e-12)
+        if coherence_check < ethical_threshold:
+            return {"status": "rejected", "reason": "Below ethical coherence threshold"}
+        
+        response = {
+            "status": "accepted",
+            "query": external_query,
+            "state": state.tolist(),
+            "intent": intent.tolist(),
+            "timestamp": datetime.now(timezone.utc).isoformat(),
+            "glyph": "🌌",
+            "notes": f"Oracle Node processed query in ICA_v1 First Light Weave",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)